procedures/observing_tools/Sky Almanac.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>State your source here... ideally it'd come out of an observatory-wide constant file (see https://github.com/lsst-ts/ts_observatory_control/tree/develop/python/lsst/ts/observatory/control/constants) but I'm not sure we have that yet.</p><p>https://github.com/lsst-ts/ts_pointing_common/blob/develop/src/data/MTPtg.info#L25 works.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>